book/src/03_5_3_mqtt.md

```diff
@@ -43,37 +43,15 @@ The `message_event` parameter in the handler closure is of type `Result<Event<Es
 Since we're only interested in processing successfully received messages, we can make use of deep pattern matching into the closure:
```

Comment:
This sentence needs changing. `message_event` is a different type now. `EspMqttEvent` and even not a Result anymore. By calling the payload method on it you get the corresponding embedded-svc EventPayload